src/main/java/com/zufar/icedlatte/astartup/ProductsReviewsAndRatingInfoUpdater.java

```diff
@@ -0,0 +1,30 @@
+package com.zufar.icedlatte.astartup;
+
+import com.zufar.icedlatte.product.entity.ProductInfo;
+import com.zufar.icedlatte.product.repository.ProductInfoRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.UUID;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class ProductsReviewsAndRatingInfoUpdater implements ApplicationRunner {
+
+    private final ProductInfoRepository productInfoRepository;
+
+    @Override
+    @Transactional
+    public void run(ApplicationArguments args) throws Exception {
+        for (ProductInfo productInfo : productInfoRepository.findAll()) {
+            UUID productId = productInfo.getProductId();
+            productInfoRepository.updateAverageRating(productId);
```

Comment:
If average rating is a column in the product table and it's updated on start up, then I guess there is no need in https://github.com/Sunagatov/Iced-Latte/blob/84d1d73739eadad1f0985b7f121033490e1c35b1/src/main/java/com/zufar/icedlatte/product/api/ProductUpdater.java#L20

Reply:
nit: also, we would need to refactor https://github.com/Sunagatov/Iced-Latte/blob/6afcd6824dba53d5399dfa325828c903367c5467/src/main/java/com/zufar/icedlatte/review/endpoint/ProductReviewEndpoint.java#L86 as fields `averageRating` and `reviewsCount` don't seem to be necessary - it's ok to be done in another PR

Reply:
Fixed